engine.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "grep",
+  "description": "grep searches for specific strings",
+  "maintainer": {
+    "name": "Alexander Mankuta  ",
```

Comment:
Will Code Climate maintain this engine moving forward?

If so, we should update this to `Code Climate` and `hello@codeclimate.com`.